.github/workflows/ci2.yml

```diff
@@ -110,8 +110,8 @@ jobs:
     - name: Build Java in docker.
       run: docker build . --file utils/docker/Dockerfile.${{ matrix.base }}
                           --build-arg DEPS_JOBS
-                          --build-arg BASE_DISTRO
-                          --build-arg DAOS_JAVA_BUILD
+                          --build-arg BASE_DISTRO=${{ matrix.with }}
```

Comment:
If no value is given here then the value from the environment will be used, which is set on L82 so the change on this line isn't required.